site/docs/v1/tech/identity-providers/twitter.adoc

```diff
@@ -37,40 +37,35 @@ Once you have logged into Twitter and have obtained access to a Twitter develope
 
 If you do not yet have a Twitter app you will need to create an app in this step. If you already have a Twitter app that you plan to use for FusionAuth please review this section anyway to ensure your configuration is correct to support the Twitter login.
 
-* https://developer.twitter.com/en/apps
+* https://developer.twitter.com/en/portal/projects-and-apps
 
-In the following screenshot you will find an example of the Create an App process. Review the following sections to ensure your configuration is correct. Fill out any required fields found on this form and click the `Create` button to create a new Twitter app.
+In the following screenshot you will find an example of the Create an App process. Review the following sections to ensure your configuration is correct.
```

Comment:
```suggestion
In the below screenshot you can see how to create an app.
```

Reply:
I'd add the screenshot here if we have one, otherwise remove the phrase "screenshot"